src/kirby/components/list/list-cell/list-cell.component.tns.html

```diff
@@ -0,0 +1,5 @@
+<FlexboxLayout class="cell" [justifyContent]="alignVertical" [alignItems]="alignHorisontal" [flexGrow]="flexGrow">
+    <Label  text="{{ lineOne }}" class="line-one" *ngIf="lineOne"></Label>
```

Comment:
Looks like your editor is set to have tabs 4 spaces long and not 2 like the rest of the solution?